no_drama/__main__.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python
+
+import argparse
+import tempfile
+import os
+import os.path
+import shutil
+import subprocess
+import zipfile
+import glob
+import fnmatch
+
+from string import Template
+from functools import partial
+
+
+this_file = os.path.realpath(__file__)
+this_directory = os.path.dirname(this_file)
+templates_dir = os.path.join(this_directory, 'templates')
+build_lib = os.path.join(this_directory, 'build_lib')
+build_files = os.path.join(this_directory, 'build_files')
+
+
+def save_wheels(destination, packages=None, requirements_file=None,
+                find_links=None):
+    call_list = ["pip", "wheel", "--wheel-dir=%s" % destination]
+
+    if packages is not None:
+        call_list += packages
+
+    if find_links is not None:
+        call_list += ['--find-links=%s' % find_links]
+
+    if requirements_file is not None:
+        call_list += ['-r', requirements_file]
+
+    subprocess.call(call_list)
+
+
+def get_file_from_dir(dir, filename, mode='r'):
+    full_path = os.path.join(dir, filename)
+    return open(full_path, mode)
+
+
+def get_template(filename, mode='r'):
+    template_file = get_file_from_dir(templates_dir, filename, mode=mode)
+    return Template(template_file.read())
+
+def stage_bundle(cli_args):
+    staging_dir = tempfile.mkdtemp()
+    build_dir = os.path.join(staging_dir, cli_args.label)
+
+    # these are wheels needed during activation
+    bootstrap_wheels = ['virtualenv','pip','wheel','distribute']
+    bootstrap_wheels_destination = os.path.join(build_dir,'bootstrap_wheels')
+    save_wheels(packages=bootstrap_wheels, destination=bootstrap_wheels_destination)
+
+    # move just the wheels we want into the bundle dir  
```

Comment:
i think this method can be broken down a bit into smaller sub-routines.

each commented block really should just be it's own function:
```
    def activate_wheels(...)
    def move_wheels_to_bundle_dir(...)
    def copy_django_proj_to_bundle_dir(...)
    def make_lib_directory(...)
    def generate_activate_shell_file(...)
```

then you can just get rid of the comments themselves.